lib/internal/per_context/primordials.js

```diff
@@ -117,7 +117,9 @@ primordials.SafePromise = makeSafe(
   'Math',
   'Reflect'
 ].forEach((name) => {
-  copyPropsRenamed(global[name], primordials, name);
+  const original = global[name];
+  primordials[name] = original;
+  copyPropsRenamed(original, primordials, name);
```

Comment:
This is needed because of <https://github.com/nodejs/node/pull/35448#discussion_r519241198>.

Reply:
Can you open a separate PR for this?

`freeze_intrinsics.js` is executed before any user code, I don't think it's worth adding it to the `primordials` object: there's no security reason to do so, IMHO we should fix `frozen_intrinsics.js` code instead.